### ISSUE TITLE
New install of ozp-center cannot load jquery

### ISSUE THREAD

**Issue by brian428** [ozone-development/ozp-center#105]
After following the getting started guide, I've started up ozp-center. If I browse to http://localhost:3000/, I get a blank page. Viewing the Chrome inspector, I see that 3 files failed to load with 404 errors: lib/jquery/jquery.min.js, lib/jquery/jquery-ui.min.js, and lib/jquery/jquery.dd.min.js.

Is this a known issue?

**Comment by wski** [comment]
Is there any specific reason you're using this version of Center? I would recommend the use of center-ui

**Comment by brian428** [comment]
No, actually I'm just trying to figure out how to set things up to test out the new platform. I couldn't find any real explanation of what the various repos here do or which one to use to get started. So I just started digging through them until I found ozp-center, which the readme led me to believe was what I was looking for.

Are you saying I should not use this, but instead use ozp-ui? Is there an explanation of which repos I need to be looking at somewhere? Thanks.

**Comment by wski** [comment]
You'll want to use the following to launch the marketplace: https://github.com/ozone-development/center-ui

also--
https://github.com/ozone-development/hud-ui
https://github.com/ozone-development/ozp-webtop
https://github.com/ozone-development/ozp-iwc
and 
https://github.com/ozone-development/ozp-rest

are going to be the core for the new platform. 

I'll dig around and see if we have a deployment guide for the new platform and report back to this issue when I find something. 

I hope this helps out a little in the mean time

**Comment by brian428** [comment]
Thanks, I'll do that. I got the center-ui running after a lot of npm problems (many of the dependencies appear to be outdated and don't play well with node 0.12, and at least one required me to go into the preprocessor module and add a manual fix). Once I got the app running, though, I'm asked for a certificate. So my next step is to figure out how to proceed there. Anyway, thanks, I'll close this and probably open one or more issues in the center-ui repo.
